CHANGELOG.md

```diff
@@ -62,7 +62,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (modules/core/04-channel) [\#1232](https://github.com/cosmos/ibc-go/pull/1232) Updating params on `NewPacketId` and moving to bottom of file.
 * (app/29-fee) [\#1305](https://github.com/cosmos/ibc-go/pull/1305) Change version string for fee module to `ics29-1`
 * (app/29-fee) [\#1341](https://github.com/cosmos/ibc-go/pull/1341) Check if the fee module is locked and if the fee module is enabled before refunding all fees
-* (modules/core/04-channel) [\#1317](https://github.com/cosmos/ibc-go/pull/1317)  Remove crossing hellos from ChanOpenTry
+* (modules/core/04-channel) [\#1317](https://github.com/cosmos/ibc-go/pull/1317) Remove corssing hellos from channel handshakes. The `PreviousChannelId` in `MsgChannelOpenTry` has been deprecated.  
```

Comment:
Since this is an API breaking change, would it make more sense to add this to the `API Breaking` section? Just to give it more visibility... I know this is a bit of a problem at the moment with the structure of our changelog...

Reply:
Sounds good!